ts/Dashboards/Board.ts

```diff
@@ -393,18 +395,22 @@ class Board implements Serializable<Board, Board.JSON> {
         }
 
         // Clear the container from any content.
-        renderTo.innerHTML = '';
-
-        // Set the main wrapper container.
-        board.boardWrapper = renderTo;
-
-        // Add container for the board.
-        board.container = createElement(
-            'div', {
-                className: Globals.classNames.boardContainer
-            }, {},
-            this.boardWrapper
-        );
+        if (this.guiEnabled) {
+            renderTo.innerHTML = '';
```

Comment:
What if there is sth in the element: https://jsfiddle.net/BlackLabel/2uaq6oed/

Reply:
There is a dashboard structure, so we cannot remove it.

Reply:
Shouldn't we also clear the elements that are not created by the GUI?

Reply:
I don't think so. There could be a layout as well. In case of declaring some cells inside the container and we will reset it, components will not be created there.